code/modules/hydroponics/grown/corn.dm

```diff
@@ -97,3 +97,28 @@
 	seed = /obj/item/seeds/corn/snapcorn
 	name = "snap corn cob"
 	desc = "A reminder of pranks gone by."
+
+/obj/item/seeds/corn/pepper
+	name = "pack of pepper-corn seeds"
+	desc = "If Peter picked a pack of pepper-corn..."
+	icon_state = "seed-peppercorn"
+	species = "peppercorn"
+	plantname = "Pepper-Corn Stalks"
+	product = /obj/item/food/grown/peppercorn
+	mutatelist = null
+	reagents_add = list(/datum/reagent/consumable/blackpepper = 0.2, /datum/reagent/consumable/nutriment/vitamin = 0.04, /datum/reagent/consumable/nutriment = 0.1)
+
+/obj/item/food/grown/peppercorn
+	seed = /obj/item/seeds/corn/pepper
+	name = "ear of pepper-peppercorn"
+	desc = "This dusty monster needs god..."
+	icon_state = "peppercorn"
+	trash_type = /obj/item/grown/corncob/pepper
+	foodtypes = VEGETABLES
+	grind_results = list(/datum/reagent/consumable/blackpepper = 0)
+	tastes = list("pepper, sneezing" = 1)
```

Comment:
```suggestion
	tastes = list("pepper" = 1, "sneezing" = 1)
```
When tasted, it'll end up being `Tastes like pepper and sneezing` or `Tastes like pepper, sneezing, etc`. So no point to combine them.